darwin/importer/importer.py

```diff
@@ -824,6 +848,11 @@ def import_annotations(  # noqa: C901
         else:
             local_files.append(parsed_file)
 
+    annotation_format = _get_annotation_format(importer)
+    _check_annotation_format_slot_compatibility(
+        local_files, remote_files, annotation_format
+    )
+
```

Comment:
It's best to check early on if the user is trying to import non-Darwin JSON 2.0 annotations to multi-slotted/multi-channeled items